stage/stage_utils.go

```diff
@@ -49,7 +49,14 @@ func (s *Stage) MergeWith(other *Stage) *Stage {
 		s.TimeZone = other.TimeZone
 	}
 	s.Queries = append(s.Queries, other.Queries...)
-	s.QueryFiles = append(s.QueryFiles, other.QueryFiles...)
+	for _, queryFile := range other.QueryFiles {
+		log.Info().Msg(fmt.Sprintf("%s||||%s", queryFile, wd))
+		absPath, err := filepath.Abs(filepath.Join(wd, queryFile))
```

Comment:
You might want to check if the query path is already an absolute path before doing `fielpath.Join()`?
In our current benchmark jsons we never used absolute path but it can be

Reply:
Do you want to do it here or in `processStagePath`